### PR TITLE
Fix hello_world.py

### PR DESCRIPTION
**Pull request by jasonrudolph**
Prior to this change, you get the following results:

```
$ python hello_world.py

Traceback (most recent call last):
  File "hello_world.py", line 1, in <module>
    echo ("Hello, World!")
NameError: name 'echo' is not defined
```

After this change, you get these results instead:

```
python hello_world.py 
Hello, World!
```

@notjasonrudolph: I hope this helps!

### CODE REVIEW

File: hello_world.py [hunk 1]
```diff
@@ -1 +1 @@
-echo ("Hello, World!")
+print "Hello, World!"
```
Comment (notjasonrudolph) [inline hello_world.py]: Thanks for this contribution, @jasonrudolph. :zap: 

While you're in here, would you mind making the greeting more personal? How about, "Hello, Python Namibia!"?

Comment (notjasonrudolph) [inline hello_world.py]: @jasonrudolph: Totally up to you, though.

**Comment by jasonrudolph** [comment]
OK, @notjasonrudolph. That change is in place now. Great suggestion! :loudspeaker: :snake:

**Comment by notjasonrudolph** [comment]
Perfect!

**Pull request merged**